homebrew_releaser/formula.py

```diff
@@ -192,6 +193,16 @@ def install
 end
 """
 
+        def match_indent_of(tag, text):
```

Comment:
suggestion: adding type hints for `tag` and `text` here along with the return type would be good to match the rest of the repo.